src/plugins/data_source_management/public/components/drop_down_header.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiTitle, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import React from 'react';
+import { ApplicationStart } from 'opensearch-dashboards/public';
+import { DSM_APP_ID } from '../plugin';
+
+interface DataSourceOptionItemProps {
+  dataSourceCount: number;
+  application?: ApplicationStart;
+}
+
+export const DataSourceDropDownHeader: React.FC<DataSourceOptionItemProps> = ({
+  dataSourceCount,
+  application,
+}) => {
+  const dataSourceCounterText = dataSourceCount === 1 ? 'DATA SOURCE' : 'DATA SOURCES';
+
+  return (
+    <EuiTitle size="xxxs">
+      <EuiFlexGroup responsive={false}>
+        <EuiFlexItem>
+          {dataSourceCounterText}({dataSourceCount})
```

Comment:
nit: There should be a space before the `(`

```suggestion
          {dataSourceCounterText} ({dataSourceCount})
```

Reply:
good catch